src/components/profile-appearance-settings.tsx

```
@@ -0,0 +1,490 @@
+
+    async function loadCooldown() {
+        try {
+            const result = await getBackgroundCooldown();
+            setCooldown(result);
+        } catch {
+            setCooldown({ canChange: true, remainingMs: 0 });
+        }
+    }
+
+    async function loadFrames() {
+        if (framesLoaded) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)